pip/src/noisy_simulator.rs

```diff
@@ -44,20 +44,22 @@ fn python_to_nalgebra_matrix(matrix: PythonMatrix) -> PyResult<SquareMatrix> {
     Ok(SquareMatrix::from_row_iterator(nrows, ncols, data))
 }
 
-fn nalgebra_matrix_to_python_list(matrix: &SquareMatrix) -> Vec<Complex<f64>> {
-    let (nrows, ncols) = matrix.shape();
-    let mut list = Vec::with_capacity(nrows * ncols);
-
+fn nalgebra_matrix_to_python_matrix(matrix: &SquareMatrix) -> PythonMatrix {
```

Comment:
Does the .pyi file need any updates for the changed signature?

Reply:
It doesn't need any updates. It was giving a 2D matrix as type hint already. I guess nobody catch it before because types in Python don't trigger any kind of error.